lib/ldp.js

```diff
@@ -1,5 +1,6 @@
 var mime = require('mime-types')
 var path = require('path')
+const url = require('url')
 var fs = require('fs')
 var $rdf = require('rdflib')
 var async = require('async')
```

Comment:
I agree (and in fact, the `async` module is sort of outdated, so I'm removing it in the LDP refactor).